tests/core/Observer.spec.ts

```diff
@@ -50,9 +51,11 @@ describe('Observer', async () => {
     downloadManager.start();
 
     const transactionProcessor = new TransactionProcessor(downloadManager, operationStore);
+    const throughputLimiter = new ThroughputLimiter(10, 25, transactionStore);
```

Comment:
For my understanding, any particular reason for choosing 10,  and 25? Doesn't look like it is required for the test to pass.

Reply:
It's just some value that also allows testing for transactions rate limiting and operation rate limiting.